CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # OverReact Changelog
 
+## 1.19.0
+
+> [Complete `1.19.0` Changeset](https://github.com/Workiva/over_react/compare/1.18.1...1.19.0)
+
+__New Features__
+* [#118] Add `BuiltReduxUiComponent` and `BuiltReduxUiProps`.
```

Comment:
We should probably mention here that these are considered unstable and found in `package:over_react/experimental.dart`